### ISSUE TITLE
Initial Pose

### ISSUE THREAD

**Issue by michellevalente** [RobustFieldAutonomyLab/LeGO-LOAM#159]
Is there any way we can provide an initial pose to the method?
Thank you in advance for the help!

**Comment by TixiaoShan** [comment]
@michellevalente 
Yes, check the function [here](https://github.com/RobustFieldAutonomyLab/LeGO-LOAM/blob/38f087c06b8832c49760d0de54ce74321f5bcc3f/LeGO-LOAM/src/featureAssociation.cpp#L1592)

Assign your initial pose to ```transformSum[]```. The order is ```pitch, yaw, roll, y, z, x``` in standard ROS frame.

**Comment by Leedk3** [comment]
Hi Shan, 
When I tried changing transformSum[] in the code, map building became worse.
Should I change other things more in the code?

The initial pose did not start from origin pose as I wanted, however, the result was like this.
If I remove "transformSum[]" things what I added, the result was no problem.  

I found that if I set the initial pose far from the origin, map drifting was being more.
In my case, when I set like this, it works:
`transformSum[3] = 10`
`transformSum[5] = 10`

However, when I set like far from the origin like this, it does not work:
`transformSum[3] = 100`
`transformSum[5] = 100`



![result](https://user-images.githubusercontent.com/37441993/95068544-63499480-0740-11eb-92bf-a1e282106e4f.png)
